aw-models/src/key_value.rs

```diff
@@ -0,0 +1,27 @@
+use chrono::{DateTime, Utc};
+
+#[derive(Serialize, Deserialize)]
+pub struct Key {
```

Comment:
I think this might be overkill, might as well just use a String wherever this is used.

Reply:
Hm yeah I actually tried to use a string, but concatenating the `{key: <key_variable>}` string together seemed much less clear and harder to write without typo's than just using this. I guess wrapping the timestamp in an option would solve this and the upper comment though.

Reply:
Alright!